Makefile

```diff
@@ -11,6 +11,8 @@ TEST_LIST_SOURCE=$(CTS_DIR)/external/vulkancts/mustpass/1.0.2/vk-default.txt
 DEQP_DIR=$(CTS_DIR)/build/external/vulkancts/modules/vulkan/
 DEQP=cd $(DEQP_DIR) && RUST_LOG=debug LD_LIBRARY_PATH=$(FULL_LIBRARY_PATH) ./deqp-vk
 CLINK_ARGS=
+COMMIT_SHA_SHORT=$(shell git rev-parse --short HEAD)
```

Comment:
I wonder if we should abort on the presence of uncommitted changes, since the commit SHA is not representative in this case?

Reply:
Yeah possibly. We've decided to include the tag now instead, not sure if it's still needed or if we should just try to be careful. Or even rename this to `travis-package` or something.